src/App.js

```diff
@@ -3,13 +3,62 @@ import Header from './Header.js';
 import Main from './Main.js';
 import Footer from './Footer.js';
 import data from './data.json';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button'
+import { CardImg, ModalBody } from 'react-bootstrap';
 
 class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selected: false,
+      image: 'placeholder',
+      blurb: 'Lorem Ipsum'
+    }
+  }
+
+
+  handleRenderModal = (imageUrl, description) => {
+    this.setState({
+      selected: true,
+      image: imageUrl,
+      blurb: description,
+    })
+  }
+
+  handleRemoveModal = () => {
+    this.setState({
+      selected: false
+    })
+  }
+
   render() {
     return (
       <>
         <Header />
-        <Main data={data}/>
+
+        <Main 
+        data={data} 
+        handleRenderModal={this.handleRenderModal}
+        />
+
+        <Modal centered show={this.state.selected} onHide={this.handleRemoveModal}>
+          <Modal.Header>
+            <Modal.Title></Modal.Title>
+          </Modal.Header>
+          <Modal.Body>
+            <img width="130%" className="zoomed"
+              src={this.state.image}
+              alt={this.state.description}
+            />
+            {/* <Button variant="primary" 
+            onClick={this.handleVote}>
+              Vote
+            </Button> */}
+            {this.state.blurb}
+          </Modal.Body>
+        </Modal>
```

Comment:
Good work on the lab so far! 

- The lab asks us to: 
  - Create a `SelectedBeast` component and include it in your `App`
- Use the state in the App to render an individual beast in a Modal in the SelectedBeast component using React Bootstrap
- For these lines, all of this will go into a `SelectedBeast` component (once you create that/update the code, we should be good) 👍